src/5e-SRD-Features.json

```diff
@@ -2025,7 +1769,11 @@
     "desc": [
       "The Circle of the Land is made up of mystics and sages who safeguard ancient knowledge and rites through a vast oral tradition. These druids meet within sacred circles of trees or standing stones to whisper primal secrets in Druidic. The circle's wisest members preside as the chief priests of communities that hold to the Old Faith and serve as advisors to the rulers of those folk. As a member of this circle, your magic is influenced by the land where you were initiated into the circle's mysterious rites."
     ],
-    "group": "Circle of the Land",
+    "parent": {
+      "index": "choose-circle-of-the-land",
```

Comment:
Ought we to update the index to match the name, i.e. removing `choose-`?

Reply:
This is also mentioned in #382 .

Reply:
> Ought we to update the index to match the name, i.e. removing `choose-`?

I agree with removing choose from the indexes. Should I do that in this pr or another one?

Reply:
If the names are changing in this one, I think the indexes should, too

Reply:
Should that include everything in https://github.com/5e-bits/5e-database/discussions/382 then as well? Or just removing choose from indexes.

Reply:
I reckon leave the rest for now. If we've decided to do the names in this PR, it feels like the indices should be done now too, just for consistency.